### PR TITLE
Update Langchains Agent, rename to Monologue

### PR DESCRIPTION
**Pull request by rbren**
A few changes in here:
* Remove langchain dependency
* Rename LanchainsAgent to MonologueAgent
* Refactor monologue initialization for the agent
* Standardize on `content` instead of `contents` (we had both)

### CODE REVIEW

File: agenthub/planner_agent/prompt.py [hunk 1]
```diff
@@ -192,9 +192,9 @@ def parse_response(response: str) -> Action:
     json_end = response.rfind("}") + 1
     response = response[json_start:json_end]
     action_dict = json.loads(response)
-    if 'content' in action_dict:
+    if 'contents' in action_dict:
         # The LLM gets confused here. Might as well be robust
-        action_dict['contents'] = action_dict.pop('content')
+        action_dict['content'] = action_dict.pop('content')
```
Comment (enyst) [inline agenthub/planner_agent/prompt.py:197]: The second of these should be 'contents', no? 😆

Comment (rbren) [reply agenthub/planner_agent/prompt.py:197]: Good catch! I must have forgotten a `/g` 😬

Comment (rbren) [inline agenthub/planner_agent/prompt.py:197]:
```suggestion
        action_dict['content'] = action_dict.pop('contents')
```

**Comment by enyst** [comment]
I ran this branch and it worked, except for the Code tab, which no longer updates with the code. It does on main.

**Comment by rbren** [comment]
Thanks for checking! I'll take a look

**Comment by rbren** [comment]
Good catch on the code editor! Found the issue, fixed now

**Pull request merged**